tests/test-core/test-init-node.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env mocha */
+
+const expect = require('chai').expect
+const IPFS = require('../../src/core')
+const IPFSRepo = require('ipfs-repo')
+
+function createTestRepo () {
```

Comment:
It would be pretty sweet to grab this func and also the logic to 'inject' a go-ipfs repo (found in index.js and browser.js) and make a utils file that can be used across tests that can 'create and teardown a go-ipfs generated repo' and 'create and teardown a fresh baked repo.

Would you like to make it on this PR too @noffle ?